erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Distributed Cost Center
+add_breadcrumbs: 1
+show_sidebar: 0
+
+metatags:
+ description: A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.
+ keywords: frappe, erpnext, accounting reports, Cost Center, GL Entry.
+---
+
+<!-- add-breadcrumbs -->
+# Distributed Cost Center
+
+**A Distributed Cost Center is a *Cost Center* in which multiple Cost Centers are tagged with an appropriate percentage.**
+
```

Comment:
@KaviyaPeriyasamy can you add an example, new readers will not understand what this, it's purpose

Reply:
@prasadarr Hope this helps newbie https://github.com/frappe/erpnext_documentation/pull/85/commits/45ef27176197f8d22a69974bdaa8c0bd8e259c78

Reply:
@KaviyaPeriyasamy it still doesn't tell me what's the benefit of having a distributed cost center. In how are the costs shared? equally? What problem does this feature solve?

Reply:
@prasadarr Check whether https://github.com/frappe/erpnext_documentation/pull/85/commits/6c0d92b833b448089d96c9fc276a07e1b0f7bb13 it clarifies?

Reply:
This is the exact thing solved by this feature.

Reply:
Thanks!

Reply:
well, just one more thing. What business problem does this solve?
i.e., why would someone want to use this feature?

Reply:
If a business has a master cost center with dependent cost centers. In every master cost center transactions, it is difficult to update the budget, profit, and loss to each dependent cost center manually with the allocated percentage of the master cost center. This feature helps to automate the process of manual entry.

Reply:
@KaviyaPeriyasamy 
please add this in the PR, I can't edit your file (tick "Allow edits from maintainers" in future PRs)

Reply:
Sure. Kindly check this 60d1fcddc082fad73a53db916c1aa09bbe8efd0f